config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -314,7 +314,108 @@ presubmits:
             memory: "16Gi"
         securityContext:
           privileged: true
-
+  # Fork of ci-kubernetes-e2e-gce-scale-performance
+  - name: ci-kubernetes-e2e-gce-scale-performance-manual
+    cluster: k8s-infra-prow-build
+    always_run: false
+    max_concurrency: 1
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-e2e-scalability-common: "true"
+      preset-e2e-scalability-periodics: "true"
+      preset-e2e-scalability-periodics-master: "true"
+    decoration_config:
+      timeout: 450m
+    extra_refs:
+    - org: kubernetes
+      repo: perf-tests
+      base_ref: master
+      path_alias: k8s.io/perf-tests
+    annotations:
+      testgrid-dashboards: presubmits-kubernetes-scalability
+      testgrid-tab-name: ci-kubernetes-e2e-gce-scale-performance-manual
+    spec:
+      volumes:
+      - name: cache-secret
+        secret:
+          secretName: scale-pull-cache-token
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250227-3a13bdd784-master
+        volumeMounts:
+        - name: cache-secret
+          readOnly: true
+          mountPath: /etc/registry-auth
+        env:
+        - name: KUBERNETES_REGISTRY_PULL_THROUGH_HOST
+          value: https://us-central1-docker.pkg.dev/v2/k8s-infra-e2e-scale-5k-project/k8s-5k-scale-cache/
+        - name: KUBERNETES_REGISTRY_PULL_THROUGH_BASIC_AUTH_TOKEN_PATH
+          value: /etc/registry-auth/token
+        command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --cluster=gce-scale-cluster
+        - --env=HEAPSTER_MACHINE_TYPE=e2-standard-32
+        # TODO(mborsz): Adjust or remove this change once we understand coredns
+        # memory usage regression.
+        - --env=KUBE_DNS_MEMORY_LIMIT=300Mi
+        - --extract=ci/fast/latest-fast
+        - --gcp-nodes=5000
+        - --gcp-project-type=scalability-scale-project
```

Comment:
There is only one of these. So running this can disrupt the periodics?

https://github.com/kubernetes/kubernetes/issues/130968

Reply:
https://github.com/kubernetes/k8s.io/blob/d13de4311167de30617555d49a3b26c5de478945/kubernetes/gke-prow-build/prow/boskos-resources-configmap.yaml#L218

Reply:
Possibly, however we don't want to increase capacity for 5k tests.

Reply:
Clarification Note: We use SIG K8s Infra managed projects now, but we still need GCP to approve quota on a project like this, and I'm also not sure if we want to either.

Reply:
I'm concerned about disrupting the release-informing periodics though.

Reply:
Was this different before removing the Job? The goal of manual scalability tests is to test high risk performance changes, like new etcd minor release. I expect we only have 1-2 features like this in release cycle.

Reply:
> Was this different before removing the Job? The goal of manual scalability tests is to test high risk performance changes, like new etcd minor release. I expect we only have 1-2 features like this in release cycle.

Looks like it:
https://github.com/kubernetes/test-infra/pull/33229/files#diff-07d9a5c0eb2050cafb795db08b673175226c577365bcfe39dc7025ab87300898L133
>         - --gcp-project-type=scalability-presubmit-5k-project

I guess we had another pool for 5k scale presubmit?

Reply:
https://github.com/kubernetes/k8s.io/issues/7929 for rolling-forward when we can

Reply:
And https://github.com/kubernetes-sigs/prow/issues/409